### PR TITLE
Simplified speech samples (set-4)

### PR DESCRIPTION
**Pull request by Thoughtseize1**
## Description

Fixes #<ISSUE-NUMBER>

Note: Before submitting a pull request, please open an issue for discussion if you are not associated with Google.

## Checklist
- [x] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md)
- [ ] README is updated to include [all relevant information](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#readme-file)
- [x] **Tests** pass:   `nox -s py-3.9` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [x] **Lint** pass:   `nox -s lint` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/CODEOWNERS) with the codeowners for this sample
- [ ] This sample adds a new **Product API**, and I updated the [Blunderbuss issue/PR auto-assigner](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/blunderbuss.yml) with the codeowners for this sample
- [ ] Please **merge** this PR for me once it is approved

### CODE REVIEW

File: speech/snippets/transcribe.py [hunk 1]
```diff
@@ -61,14 +60,20 @@ def transcribe_file(speech_file: str) -> speech.RecognizeResponse:
 
 
 # [START speech_transcribe_sync_gcs]
-def transcribe_gcs(gcs_uri: str) -> speech.RecognizeResponse:
-    """Transcribes the audio file specified by the gcs_uri."""
+def transcribe_gcs(audio_uri: str) -> speech.RecognizeResponse:
+    """Transcribes the audio file specified by the gcs_uri.
+    Args:
+        audio_uri (str): The Google Cloud Storage URI of the input audio file.
+            E.g., gs://[BUCKET]/[FILE]
```
Comment (msampathkumar) [inline speech/snippets/transcribe.py:67]:
```suggestion
            E.g., gs://cloud-samples-data/speech/audio.flac
```

File: speech/snippets/transcribe_multichannel.py [hunk 2]
```diff
@@ -64,22 +56,20 @@ def transcribe_file_with_multichannel(speech_file: str) -> speech.RecognizeRespo
...
-    """Transcribe the given audio file on GCS with
-    multi channel.
-
+def transcribe_gcs_with_multichannel(audio_uri: str) -> speech.RecognizeResponse:
+    """Transcribe the given audio file from Google Cloud Storage synchronously with multichannel.
     Args:
-        gcs_uri: A path to audio file to be recognized.
-
+        audio_uri (str): The Cloud Storage URI of the input audio.
+            E.g., gs://[BUCKET]/[FILE]
```
Comment (msampathkumar) [inline speech/snippets/transcribe_multichannel.py:63]:
```suggestion
            E.g., gs://cloud-samples-data/speech/multi.wav
```

**Review by msampathkumar — COMMENTED** [review]
Need minor doc changes. Check provided suggestions.

File: speech/snippets/transcribe.py (hunk 1 above)
Comment (Thoughtseize1) [reply speech/snippets/transcribe.py:67]: Done✅

File: speech/snippets/transcribe_multichannel.py (hunk 2 above)
Comment (Thoughtseize1) [reply speech/snippets/transcribe_multichannel.py:63]: Done✅

**Review by msampathkumar — APPROVED** [review]
LGTM

**Pull request merged**